### ISSUE TITLE
Delete doc2vec.d2v and corpus-vectors.pkl in experiments/demo 

### ISSUE THREAD

**Issue by sverhoeven** [ADAH-EviDENce/evidence#121]
Can in experiments/demo the doc2vec.d2v and corpus-vectors.pkl removed from repo? They are produced by model generation but not used by frontend.

**Comment by meiertgrootes** [comment]
Yes, they can be removed I guess. It looked like someone had left them in by choice, so I hadn't removed them, but I think they can go.
